docs/src/main/sphinx/release/release-438.md

```diff
@@ -0,0 +1,36 @@
+# Release 438 (31 Jan 2024)
+
+## General
+
+* Add support for using string types with the {func}`reduce_agg` function. ({issue}`20452`)
+* Fix query failure when using `char` types with the {func}`reverse` function. ({issue}`20387`)
+
+## Delta Lake connector
+
+* Add support for configuring the maximum value count for pages written in
```

Comment:
what is a "value count"

maybe 

the maximum pages

or 

the maximum number of pages

also the property name is similarly bad imho @martint 

and same change for all others

Reply:
I think it's the number of values that can be on a page, not the number of pages.

Reply:
maybe.. its not clear to me ..

Reply:
I will let @martint weigh in and decide.

Reply:
This is a very low-level config. It appears to control the number of values that go in a parquet page within a column chunk. 

The current phrasing is the best we can do. Maybe change the grammar slightly to say: "the maximum number of values per page when writing to Parquet files"